lib/src/navigation.dart

```diff
@@ -98,6 +100,11 @@ final watchScrollController = ScrollController(debugLabel: 'WatchScroll');
 final RouteObserver<PageRoute<void>> rootNavPageRouteObserver =
     RouteObserver<PageRoute<void>>();
 
+final cupertinoTabControllerProvider =
```

Comment:
Not really useful to declare a state provider here, you could just use a private `final` variable.